includes/admin/settings/class-settings-bootstrap.php

```diff
@@ -4,36 +4,50 @@
  */
 final class WPBDP__Settings__Bootstrap {
 
-    public static function register_initial_groups() {
-        wpbdp_register_settings_group( 'general', _x( 'General', 'settings', 'business-directory-plugin' ) );
-
-        wpbdp_register_settings_group( 'listings', _x( 'Listings', 'settings', 'business-directory-plugin' ) );
-        wpbdp_register_settings_group( 'listings/main', _x( 'General Settings', 'settings', 'business-directory-plugin' ), 'listings' );
-
-        wpbdp_register_settings_group( 'email', __( 'Email', 'business-directory-plugin' ) );
-        wpbdp_register_settings_group( 'email/main', _x( 'General Settings', 'settings', 'business-directory-plugin' ), 'email' );
-
-        wpbdp_register_settings_group( 'payment', _x( 'Payment', 'settings', 'business-directory-plugin' ) );
-        wpbdp_register_settings_group( 'payment/main', _x( 'General Settings', 'settings', 'business-directory-plugin' ), 'payment' );
-
-        wpbdp_register_settings_group( 'appearance', _x( 'Appearance', 'settings', 'business-directory-plugin' ) );
-
-        wpbdp_register_settings_group( 'modules', __( 'Module Settings', 'business-directory-plugin' ) );
-
-		wpbdp_register_settings_group( 'misc', __( 'Miscellaneous', 'business-directory-plugin' ) );
-    }
+	public static function register_initial_groups() {
```

Comment:
The diff in this file is too big also. Can you please revert the tabs/spaces for lines that aren't getting changed?

Reply:
Spacing reverted in this file

Reply:
Nevermind! This one is great now.